make-release.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <release_version>"
+    exit 1
+fi
+
+RELEASE_VERSION=$1
+
+# Update CHANGES.md
+sed -i "s/## Unreleased/&\n\n## $RELEASE_VERSION\n\n%RELEASE NOTES TEMPLATE - Lines starting with '%' will be ignored.\n%This description should answer the \"who needs this update?\" question.\n/" CHANGES.md
```

Comment:
This description should answer "who needs this update?"

Can you try splitting the sed argument into multiple lines?